deployment/terraform/cdn.tf

```diff
@@ -61,6 +61,27 @@ resource "aws_cloudfront_distribution" "cdn" {
     max_ttl                = 300
   }
 
+  ordered_cache_behavior {
+    path_pattern     = "tile/*"
+    allowed_methods  = ["GET", "HEAD", "OPTIONS"]
+    cached_methods   = ["GET", "HEAD", "OPTIONS"]
+    target_origin_id = "originAlb"
+
+    forwarded_values {
+      query_string = false
```

Comment:
I think that we want this to be `true`. With this set to `false`, query string parameters won't get included as part of CloudFront's cache key (e.g., `tile/1?q=foo` and `tile/1?q=bar` will be processed as `tile/1`).